app/views/videos/show.html.haml

```diff
@@ -11,3 +11,34 @@
         .actions
           %a.btn.btn-primary(href="") Watch Now
           %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map{|number| [pluralize(number, "star")]}), class: "form-control"
```

Comment:
`[5,4,3,2,1].map{|number| [pluralize(number, "star")]}` this part has 1 issue.

HOMEWORK: Try to find out what is the issue.
Hint: you should look at produced HTML code and the value of option tags.

Also I would recommend to refactor it to a helper method.